test/file.js

```diff
@@ -8,21 +8,24 @@ const assert = require('assert');
 
 describe('/file', () => {
 
+  beforeEach(() => {
+    // delete the require cache
+    delete require.cache[require.resolve('../app')];
+    delete require.cache[require.resolve('config')];
+    delete require.cache[require.resolve('../controllers/file')];
+  });
+
   describe('config', () => {
-    it('returns an error if the default password isnt changed', () => {
+    it('returns an error if the default password isnt set', () => {
+      process.env.NODE_CONFIG = '{"aws": {"password":""}}';
+
       assert.throws(() => require('../controllers/file'), Error, 'please set the AWS_PASSWORD');
+
+      process.env.NODE_CONFIG = '{"aws": {"password":"atest"}}';
```

Comment:
This would be better done in a `beforeEach`/`afterEach`. As it is `process.env` won't be reset if the test fails.

Reply:
nice

Reply:
will change that

Reply:
updated